specific/cmo_project/models/project.py

```diff
@@ -293,15 +315,116 @@ def _get_state_before_inactive(self):
     @api.multi
     def _set_project_analytic_account(self):
         for project in self:
-            parent_project = self.env['project.project'].browse(project.project_parent_id.id)
+            parent_project = self.env['project.project'].browse(
+                project.project_parent_id.id)
             project.parent_id = parent_project.analytic_account_id.id
```

Comment:
I am not sure about this.
Is "parent_id" and analytic_account? If so, it is ok.
Also,
Normally you don't need .id to assign m2o to m2o. , i..e, `project.location_id = purchase.order_line.locatoin_id`